### DISCUSSION TITLE
hexo通过git推送成功，但服务器不更新

### DISCUSSION THREAD

**Discussion by jubaofang** [hexojs/hexo#5466] [Question / Answer]
hexo推送显示INFO  Deploy done: git，但是服务器public文件并没有更新，请问这一般是哪里配置不对？

**Comment by uiolee** [comment]
You haven't provided any valid information and no one can help you. 

If you really need help, please provide more context and complete error log.

**Comment by jubaofang** [comment]
来信已经收到，这是一条自动回复提示！

**Comment by jubaofang** [comment]
来信已经收到，这是一条自动回复提示！
